src/redux/data.js

```diff
@@ -420,7 +420,7 @@ export const assetPricesReceived = message => dispatch => {
 };
 
 export const assetPricesChanged = message => (dispatch, getState) => {
-  const price = get(message, 'payload.prices[0]');
+  const price = get(message, 'payload.prices[0].price');
```

Comment:
Just FYI: the prices array updates with the entire asset object, so we just need to grab the price.